src/config/env/default.js

```diff
@@ -28,6 +28,9 @@ const config = {
   debug: false,
   SENDGRID_API_KEY: process.env.SENDGRID_API_KEY,
   FRONTEND_URL: process.env.FRONTEND_URL,
+  TWILIO_ACCOUNT_SID: process.env.TWILIO_ACCOUNT_SID,
+  TWILIO_AUTH_TOKEN: process.env.TWILIO_AUTH_TOKEN,
+  TWILIO_PHONENUMBER: process.env.TWILIO_PHONENUMBER,
```

Comment:
What if we can group these variables in a named property like (for more clarity):
```
  twilioConfig: {
    TWILIO_ACCOUNT_SID: process.env.TWILIO_ACCOUNT_SID,
    TWILIO_AUTH_TOKEN: process.env.TWILIO_AUTH_TOKEN,
    TWILIO_PHONENUMBER: process.env.TWILIO_PHONENUMBER,
  }
```